src/scss/components/_testimony.scss

```diff
@@ -0,0 +1,56 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.testimony.css Testimony
+  *
+  * ## Services
+  * @demo demo/gaiden-css/components/testimony/demo.html
+**/
+.wrap__testimony__itens {
+  float: left;
+}
+
+.testimony {
+  justify-content: center;
+
+  &__quote {
+    padding: 44px 20px;
+    position: relative;
+    background-color: get-color(air);
+    border-radius: 10px;
+
+    &::after {
+      position: absolute;
+      bottom: -29px;
+      left: 50%;
+      border-top: 15px solid get-color(air);
+      border-right: 15px solid get-color(breeze);
+      border-bottom: 15px solid get-color(breeze);
+      border-left: 15px solid get-color(breeze);
+      transform: translateX(-100%);
+      content: '';
+    }
+  }
+
+  &__user {
+    &__photo {
+      width: 30px;
+      height: 30px;
+      float: left;
```

Comment:
@MarcoBrunoBR we are using flexbox to organize our elements. We can use the modifier `--no-flexbox` to cover older browsers, but not as default property.